src/index.js

```diff
@@ -1,48 +1,77 @@
 import './style.css';
+/* eslint-disable no-console */
+/* eslint-disable arrow-body-style */
```

Comment:
- [ ] I dont think you should disable any of the `eslint` style guides since they enhance quality and clean code.